packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/Servers.tsx

```diff
@@ -65,46 +69,45 @@ function ServerList(props: State) {
         showSearchBar={true}
         disableSearch={disabledRows}
       />
-      <Table
-        fetching={{
-          fetchStatus: fetchAttempt.status === 'processing' ? 'loading' : '',
-        }}
-        columns={[
-          {
-            key: 'hostname',
-            headerText: 'Hostname',
-            isSortable: true,
-          },
-          {
-            key: 'addr',
-            headerText: 'Address',
-            isSortable: true,
-            render: renderAddressCell,
-          },
-          {
-            key: 'labelsList',
-            headerText: 'Labels',
-            render: ({ labelsList }) => (
-              <ClickableLabelCell
-                labels={labelsList}
-                onClick={onAgentLabelClick}
-              />
-            ),
-          },
-          {
-            altKey: 'connect-btn',
-            render: server =>
-              renderConnectCell(
-                () => getSshLogins(server.uri),
-                login => connect(server.uri, login)
+      <Wrapper className={disabledRows ? 'disabled' : ''}>
```

Comment:
I suppose renaming `Wrapper` to something that more directly expresses the role that this element plays would be a good idea!

Reply:
`FetchStatusBox` ?? Can't really think of a good name here. I'm open to whatever. It's purpose is to darken things while fetching lol
`FetchDarkener` sounds kinda fantasy and thats neat

Reply:
I like that!